src/api/app/controllers/webui/users/bs_requests_controller.rb

```diff
@@ -4,7 +4,10 @@ class BsRequestsController < WebuiController
       before_action :check_display_user
 
       def index
-        @requests_data_table = BsRequest::DataTable.new(params, @displayed_user)
+        parsed_params = BsRequest::DataTable::ParamsParser.new(params).parsed_params
+        requests_query = BsRequest::DataTable::FindForUserQuery.new(@displayed_user, parsed_params)
+        @requests_data_table = BsRequest::DataTable::Table.new(requests_query, parsed_params[:draw])
```

Comment:
Its the result of the [`ParamsParser` ](https://github.com/evanrolfe/open-build-service/blob/9cc0ef71f0a8b2fec5a38a3742aaffb34be90a07/src/api/app/models/bs_request/data_table/params_parser.rb)class being applied to the raw request params. It converts the params given to us from the js datatables plugin to parsed_params which can be passed to `FindForUserQuery`